### PR TITLE
Inbound sync API changes

### PR DESCRIPTION
**Pull request by dhanyahk**

### CODE REVIEW

**Comment by dhanyahk** [comment]
Made changes to synchronization API to include inbound flows as well

File: api-reference/beta/resources/synchronization-overview.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 title: "Azure AD synchronization API overview"
-description: "Automate the creation, maintenance, and removal of identities in cloud applications."
+description: "Automate the creation, maintenance, and removal of identities from AD to AAD, HR apps (Workday) to AAD and from AAD to cloud applications."
 localization_priority: Normal
 doc_type: conceptualPageType
-author: "ArvindHarinder1"
+author: "ArvindHarinder1; dhanyahk"
```
Comment (davidmu1) [inline api-reference/beta/resources/synchronization-overview.md:6]: I believe there should only be one author.

File: api-reference/beta/resources/synchronization-overview.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 title: "Azure AD synchronization API overview"
-description: "Automate the creation, maintenance, and removal of identities in cloud applications."
+description: "Automate the creation, maintenance, and removal of identities from AD to AAD, HR apps (Workday) to AAD and from AAD to cloud applications."
```
Comment (davidmu1) [inline api-reference/beta/resources/synchronization-overview.md:3]: I would keep this description simpler and provide more in the introduction paragraph.

Automate the creation, maintenance, and removal of identities.

File: api-reference/beta/resources/synchronization-overview.md [hunk 3]
```diff
@@ -13,7 +13,12 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Azure Active Directory (Azure AD) identity synchronization (also called "provisioning") allows you to automate the creation, maintenance, and removal of identities in cloud (software as a service, or SaaS) applications such as Dropbox, Salesforce, ServiceNow, and more. You can use the synchronization APIs in Microsoft Graph to manage identity synchronization programmatically, including:
+Azure Active Directory (Azure AD) identity synchronization (also called "provisioning") allows you to automate the provision (creation, maintenance) and de-provision (removal) of identities from:
```
Comment (davidmu1) [inline api-reference/beta/resources/synchronization-overview.md:16]: Azure Active Directory (Azure AD) identity synchronization (also called "provisioning") allows you to automate the provisioning (creation, maintenance) and deprovisioning (removal) of identities from:

File: api-reference/beta/resources/synchronization-overview.md [hunk 4]
```diff
@@ -13,7 +13,12 @@ Namespace: microsoft.graph
...
+- Active Directory to Azure Active Directory 
+- Workday to Azure Active Directory 
+- Azure Active Directory to cloud (software as a service, or SaaS) applications such as Dropbox, Salesforce, ServiceNow, and more.
```
Comment (davidmu1) [inline api-reference/beta/resources/synchronization-overview.md:19]: Are users going to be able to identify the "and more" applications?

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+---
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md:1]: This file looks like it is named incorrectly: reportroot-ist-credentialusagesummary.md

Should it be: reportroot-list-credentialusagesummary.md

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+author: dkershaw
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md:2]: Is Dan really the owner of this topic?

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md (hunk 5 above)
Comment (dhanyahk) [reply api-reference/beta/api/reportroot-ist-credentialusagesummary.md:1]: I need to delete these unnecessary files. I don't how it got here in this PR

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+localization_priority: Normal
+ms.prod: identity and access reports
+ms.date: 04/25/2019
+---
+# Credential Usage Summary
+Provides the summary of self-service password reset usage for a given tenant. This API provides the current state of how many users in your organization are using self-service password reset capabilities
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md:8]: Add a period to the end of the sentence.

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+|Application                            |Reports.Read.All |
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET /reports/credentialUsageSummary
+
+```
+## Optional query parameters
+This method supports the [OData Query Parameters](http://graph.microsoft.io/docs/overview/query_parameters) to help customize the response.
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md:26]: This method supports the [OData Query Parameters](http://graph.microsoft.io/docs/overview/query_parameters) listed in the following table to help customize the response.

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+|userPrincipalName|	Filter by User Principal Name|	/reports/userCredentialUsageDetails?$filter=userPrincipalNameeq 'ABCD' |	Eq, startswith(), orderby. Support case insensitive|
+|isSuccess|	Filter by status of the activity|	/reports/userCredentialUsageDetails?$filter=isSuccess eq true	|	Eq, orderby|
+|authMethod	 |Filter by the authentication methods using during registration|	/reports/userCredentialUsageDetails?$filter=authMethod eq Microsoft.AAD.Reporting.authMethodsType'email'	|Eq, orderby|
+|failureReason	|Filter by failure reason (if the activity has failed)|	/reports/userCredentialUsageDetails?$filter=failureReason eq 'ABCD'|Eq, startswith(), orderby. Support case insensitive|
+
+## Request headers
+
+| Name      |Description|
+|:----------|:----------|
+| Authorization | Bearer {code} |
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md:40]: Bearer {token}

File: api-reference/beta/api/reportroot-list-credentialuserregistrationcount.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
+---
+author: dkershaw
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-credentialuserregistrationcount.md:2]: Should this really be Dan?

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
+---
+author: dkershaw
+localization_priority: Normal
+ms.prod: identity and access reports
+ms.date: 04/25/2019
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: ms.date isn't a metadata value used in the api pages.

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
+---
+author: dkershaw
+localization_priority: Normal
+ms.prod: identity and access reports
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: This should be ms.prod: "reports"

And there should also be a metadata property added:

doc_type: "apiPageType"

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+<!-- { "blockType": "ignored" } -->
+```http
+GET /reports/credentialUsageSummary
+
+```
+## Optional query parameters
+This method supports the [OData Query Parameters](http://graph.microsoft.io/docs/overview/query_parameters) to help customize the response.
+| Name      |Description|Example|Supported Operators
+|:----------|:----------|:-------|:-------------------|
+|feature|	Filter by type of usage data you want (registration vs.reset)|	/reports/userCredentialUsageDetails?$filter=feature eq Microsoft.AAD.Reporting.featureType'registration'|eq|
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: Filter by type of usage data that you want to see, such as registration or reset activity.

File: api-reference/beta/resources/synchronization-overview.md (hunk 4 above)
Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md:19]: this doc has been there forever. So, am assuming yes

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+|userDisplayName|Filter by UserName|/reports/userCredentialUsageDetails?$filter=userDisplayName eq 'ABCD'|Eq, startswith(), orderby. Support case insensitive|
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: Filter by using the user name.

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+|userPrincipalName|	Filter by User Principal Name|	/reports/userCredentialUsageDetails?$filter=userPrincipalNameeq 'ABCD' |	Eq, startswith(), orderby. Support case insensitive|
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: Filter by user principal name.

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+|isSuccess|	Filter by status of the activity|	/reports/userCredentialUsageDetails?$filter=isSuccess eq true	|	Eq, orderby|
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: Filter by status of the activity.

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 17]
```diff
@@ -0,0 +1,98 @@
...
+|authMethod	 |Filter by the authentication methods using during registration|	/reports/userCredentialUsageDetails?$filter=authMethod eq Microsoft.AAD.Reporting.authMethodsType'email'	|Eq, orderby|
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: Filter by the authentication methods used during registration.

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+|failureReason	|Filter by failure reason (if the activity has failed)|	/reports/userCredentialUsageDetails?$filter=failureReason eq 'ABCD'|Eq, startswith(), orderby. Support case insensitive|
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: If the activity has failed, filter by failure reason.

File: api-reference/beta/api/reportroot-list-credentialuserregistrationcount.md [hunk 19]
```diff
@@ -0,0 +1,86 @@
...
+localization_priority: Normal
+ms.prod: identity and access reports
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-credentialuserregistrationcount.md]: "reports"

Add:

doc_type: "apiPageType"

File: api-reference/beta/api/reportroot-list-credentialuserregistrationcount.md [hunk 20]
```diff
@@ -0,0 +1,86 @@
...
+ms.date: 04/25/2019
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-credentialuserregistrationcount.md]: Remove ms.date.

File: api-reference/beta/api/reportroot-ist-credentialusagesummary.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+author: dkershaw
+localization_priority: Normal
+ms.prod: identity and access reports
+ms.date: 04/25/2019
+---
+# Credential Usage Summary
+Provides the summary of self-service password reset usage for a given tenant. This API provides the current state of how many users in your organization are using self-service password reset capabilities
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-ist-credentialusagesummary.md]: ../../../concepts/permissions_reference.md

Should be:

/graph/permissions-reference

File: api-reference/beta/api/reportroot-list-credentialuserregistrationcount.md [hunk 22]
```diff
@@ -0,0 +1,86 @@
...
+# Credential User Registration Summary
+
+Provides the summary of self-service password reset and multi-factor authentication (MFA) registration for a given tenant. This API provides the current state of how many users in your organization are registered for self-service password reset and multi-factor authentication (MFA) capabilities.
+
+
+Retrieve the properties and relationships of credentialuserregistrationcount object.
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-credentialuserregistrationcount.md]: ../../../concepts/permissions_reference.md

Should be:

/graph/permissions-reference

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 23]
```diff
@@ -1,4 +1,5 @@
 ---
+<<<<<<< HEAD
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: I don't think this should be here.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 24]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
 The following is an example of the request.
 
 # [HTTP](#tab/http)
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: Not sure what is going on here, but it doesn't look right.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 25]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+=======
+author: dkershaw
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: Should this really be Dan?

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 26]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+localization_priority: Normal
+ms.prod: identity and access reports
+ms.date: 04/25/2019
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: remove ms.date

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 27]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
 The following is an example of the request.
 
 # [HTTP](#tab/http)
+=======
+author: dkershaw
+localization_priority: Normal
+ms.prod: identity and access reports
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: "reports"

Add:

doc_type: "apiPageType"

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 28]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+ms.date: 04/25/2019
+---
+
+# List Credentials Usage Details 
+
+Provides the details of self-service password reset usage for a given tenant. Details include User, status of the reset, reason for failure etc.
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: I would remove this sentence:

Details include User, status of the reset, reason for failure etc.

etc. is very rarely helpful.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 29]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+|Delegated (personal Microsoft account) | Not supported. |
+|Application                            |Reports.Read.All |
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET /reports/userCredentialUsageDetails
+```
+## Optional query parameters
+This method supports the [OData Query Parameters](http://graph.microsoft.io/docs/overview/query_parameters) to help customize the response.
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: This method supports the OData Query Parameters listed in the following table to help customize the response.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 30]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+
+| Name      |Description|Example|Supported Operators|
+|:----------|:----------|:-------|:-------------------|
+|Period|Filter using time period for which you need the usage data|/reports/getCredentialUsageSummary(period='D30') /reports/getCredentialUsageSummary(period='d30')|D1, D7,D30. Period is case insensitive
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: Filter using the time period for which you need the usage data.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 31]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+|feature|Filter by type of usage data you want (registration vs. reset)|/reports/getCredentialUsageSummary(period='D30') ?$filter=feature eq Microsoft.AAD.Reporting.featureType'registration'|Eq|
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: Filter by the type of usage data that you want to see, such as registration or reset activities.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 32]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+
+
+## Request headers
+| Name      |Description|
+|:----------|:----------|
+| Authorization | Bearer {code} |
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: Bearer {token}

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 33]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+
+## Response
+If successful, this method returns a `200 OK` response code and collection of [userCredentialUsageDetails](../resources/usercredentialusagedetails.md) objects in the response body.
+## Example
+##### Request
+The following is an example of the request.
+>>>>>>> master
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: >>>>>>> master

Should not be there.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 34]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+
+## Request headers
+| Name      |Description|
+|:----------|:----------|
+| Authorization | Bearer {code} |
+
+## Response
+If successful, this method returns a `200 OK` response code and collection of [userCredentialUsageDetails](../resources/usercredentialusagedetails.md) objects in the response body.
+## Example
+##### Request
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: this should be an H3 not an H4: `### Request`

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 35]
```diff
@@ -71,10 +72,56 @@ The following example shows how to call this API.
...
+The following is an example of the request.
+>>>>>>> master
 <!-- {
   "blockType": "request",
   "name": "get_usercredentialusagedetails"
 }-->
+<<<<<<< HEAD
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: `<<<<<<< HEAD`

Should not be there.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 36]
```diff
@@ -100,37 +147,67 @@ The following is an example of the response.
 
 > **Note:** The response object shown here might be shortened for readability. All the properties are returned from an actual call.
 
+=======
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: `========`

Should not be there.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 37]
```diff
@@ -100,37 +147,67 @@ The following is an example of the response.
...
+```http
+GET https://graph.microsoft.com/beta/reports/userCredentialUsageDetails
+```
+##### Response
+The following is an example of the response. Note: The response object shown here may be truncated for brevity. All of the properties will be returned from an actual call.
+>>>>>>> master
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: Shouldn't be there.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 38]
```diff
@@ -100,37 +147,67 @@ The following is an example of the response.
...
 <!-- {
   "blockType": "response",
   "truncated": true,
   "@odata.type": "microsoft.graph.userCredentialUsageDetails",
   "isCollection": true
 } -->
+<<<<<<< HEAD
 
+=======
+>>>>>>> master
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: These three lines should not be here.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 39]
```diff
@@ -100,37 +147,67 @@ The following is an example of the response.
...
 ```http
 HTTP/1.1 200 OK
 Content-type: application/json
 Content-length: 258
 
+<<<<<<< HEAD
+=======
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: These two lines shouldn't be there.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 40]
```diff
@@ -100,37 +147,67 @@ The following is an example of the response.
...
+Content-Type: application/json
+>>>>>>> master
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: This line shouldn't be there.

File: api-reference/beta/api/reportroot-list-usercredentialusagedetails.md [hunk 41]
```diff
@@ -100,37 +147,67 @@ The following is an example of the response.
...
 }
 ```
 
+<<<<<<< HEAD
 <!-- uuid: 16cd6b66-4b1a-43a1-adaf-3a886856ed98
 2019-02-04 14:57:30 UTC -->
+=======
+<!-- uuid: 8fcb5dbc-d5aa-4681-8e31-b001d5168d79
+2015-10-25 14:57:30 UTC -->
+>>>>>>> master
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot-list-usercredentialusagedetails.md]: Same comment for all of these merge conflict markers.

File: api-reference/beta/api/reportroot_list_credentialuserregistrationdetails.md [hunk 42]
```diff
@@ -0,0 +1,93 @@
+---
+author: dkershaw
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot_list_credentialuserregistrationdetails.md]: I don't think this should be Dan.

File: api-reference/beta/api/reportroot_list_credentialuserregistrationdetails.md [hunk 43]
```diff
@@ -0,0 +1,93 @@
...
+localization_priority: Normal
+ms.prod: identity and access reports
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot_list_credentialuserregistrationdetails.md]: "reports"

Add:

doc_type: "apiPageType"

File: api-reference/beta/api/reportroot_list_credentialuserregistrationdetails.md [hunk 44]
```diff
@@ -0,0 +1,93 @@
...
+ms.date: 04/25/2019
```
Comment (davidmu1) [inline api-reference/beta/api/reportroot_list_credentialuserregistrationdetails.md]: Remove ms.date

**Pull request closed without merging**